<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Cargar_Dataset">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>421</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar Dataset</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Separadores">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Separadores</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Coma">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Coma (,)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Pipe">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pipe (|)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_P_Coma">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Punto y coma (;)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Hashtag">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hashtag (#)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Dos_Puntos">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dos Puntos (:)</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Dataset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>901</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Buscar_Dataset">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar Dataset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ruta_Dataset">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ruta Dataset</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Test_Entrenamiento">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrenamiento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Vector">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>821</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Vector">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>560</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>obtener vector</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
